src/pages/layer-2.js

```diff
@@ -0,0 +1,1084 @@
+                />
+              )
+            })}
+        </CardGrid>
+      </PaddedContent>
+
+      <PaddedContent>
+        <h2>Application specific layer 2s</h2>
+        <p>
+          Application specific layer 2s are projects that specialize in
+          optimizing for a specific application space, bringing improved
+          performance.
+        </p>
+        <CardGrid>
+          {layer2DataCombined
+            .filter((l2) => l2.purpose.indexOf("universal"))
+            .map((l2, idx) => {
+              return (
+                <Layer2ProductCard
+                  key={idx}
+                  background={l2.background}
+                  image={getImage(data[l2.imageKey])}
+                  description={l2.description}
+                  url={l2.website}
+                  note={translateMessageId(l2.noteKey, intl)}
+                  name={l2.name}
+                  bridge={l2.bridge}
+                  ecosystemPortal={l2.ecosystemPortal}
+                  tokenLists={l2.tokenLists}
+                >
+                  {l2.purpose.map((purpose) => (
+                    <Pill>{purpose}</Pill>
+                  ))}
+                </Layer2ProductCard>
+              )
+            })}
+        </CardGrid>
+      </PaddedContent>
+
+      <PaddedContent>
+        <h2>A note on alt L1s, sidechains, and validiums</h2>
+        <TwoColumnContent>
+          <Flex50>
+            <p>
+              <b>Alternative layer 1s</b> have higher throughput and lower
+              transaction fees than Ethereum. These alt L1s have had to{" "}
+              <b>sacrifice on security or decentralization</b> in order to
+              achieve higher transactions per second and lower fees. The
+              Ethereum ecosystem is firmly aligned that{" "}
+              <b>
+                layer 2 scaling is the only way to solve the scalability
+                trilemma
+              </b>{" "}
+              while remaining decentralized and secure.
+            </p>
+          </Flex50>
+          <Flex50>
+            <p>
+              <b>Sidechains and validiums</b> are blockchains that allow assets
+              from one blockchain to be bridged over and used on another
+              blockchain. Sidechains and validiums run in parallel with the main
+              chain, and interact with the main chain through bridges, but they
+              do not derive their security or data availability from the main
+              chain. They scale similarly to layer 2s, but have different trust
+              assumptions. They offer lower transaction fees, and higher
+              transaction throughput. More on{" "}
+              <Link to="/developers/docs/scaling/sidechains/">sidechains</Link>{" "}
+              and <Link to="/developers/docs/scaling/validium/">validiums</Link>
+              .
+            </p>
+          </Flex50>
+        </TwoColumnContent>
+      </PaddedContent>
```

Comment:
I feel like having these side by side with no clear separation isn't the easiest to read. I'll comment on Figma.

<img width="1603" alt="Screenshot 2022-03-30 at 18 31 48" src="https://user-images.githubusercontent.com/62268199/160896234-f68dbb3e-fd25-4c8b-bcdc-399cb8a784af.png">